MAINT: remove deprecation warning for **kwargs (#198)
adriangb/scikeras: docs/source/migration.rst

.. code:: python

   clf = KerasClassifier(..., fit__batch_size=32, predict__batch_size=10000)

If you want to change the parameters on a live instance, you can do:

.. code:: python

   clf = KerasClassifier(...)
   clf.set_params(fit__batch_size=32, predict__batch_size=10000)
   clf.fit(...)

> This effectively gives you the same result as passing ``**kwargs``, just with

Reviewer: ```suggestion
Functionally, this is the same as passing these parameters during initialization, just with
```

Author: I am specifically trying to give users a way that they can do this if they do _not_ want to pass the parameters via initialization, hence the comparison to passing `**kwargs`. The first part looks good though.

Reviewer: I'm confused by your statement. I only tried to replace "`**kwargs`" with words. Wouldn't you pass `**kwargs` to `__init__`?

Author: Sorry if it was confusing. What I meant is that this should be the same as passing `**kwargs` to `fit`. So combining my thoughts and yours, I think it would look something like:
```
Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.
```

Reviewer: > Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.

Yeah, I think that's a good representation of the message you were trying to convey. I might add a couple words about how it's better because it follows the Sciikit-learn API.

Author: Edited to:

```
Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.
This is much more compatible with the Scikit-Learn API.
In fact, this is what Scikti-Learn does in the background for hyperparameter tuning.
```
